onnxscript/tests/function_libs/torch_lib/error_reproduction.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import annotations
+
+import pathlib
+import platform
+import sys
+import time
+import traceback
+from typing import Any, Mapping
+
+import numpy as np
+import onnx
+import onnxruntime as ort
+import torch
+
+_REPRODUCTION_TEMPLATE = '''\
```

Comment:
Any consideration using jinja or libcst for codegen?

Reply:
I didn’t want to take jinja as an extra dependency. As for libcst I think it’s overkill and obscures the actual generated code at the moment